vite.config.js

```diff
@@ -13,6 +15,9 @@ export default function config({ mode }) {
       mdx.default({
         remarkPlugins: [slug, autoLinkHeadings, jsxExample, propTable],
       }),
+      uno({
+        presets: [presetWarp({ usePixels: true, usePreflight: true })]
```

Comment:
Just curious why this is using pixels? Does storybook break with the REM root change?

Reply:
It kind of broke, if I remember it correctly. This was the easy way forward, I guess, as Fabric/react used pixels? @BalbinaK?
But we should change to using rem from now on?

Reply:
We definitely should use REM if possible, and if not possible using Storybook that might put a stronger emphasis on moving to Ladle (if it would support REM better).

This environment is where we're developing and ensuring components are behaving as they should - and part of that is making sure accessibility features like resizes work properly.

Using pixels will break that, and make it rather difficult to ensure React components behave as they will in real world scenarios. Unless ofc y'all have some workaround in mind to ensure this behavior works.

Reply:
Yea I agree, but not a shostopper for this pr, is it ? :)

Reply:
We'll merge this and then have another look at changing it to rem. 👍